JSDemos/Demos/ButtonGroup/Overview/description.md

```diff
@@ -1,3 +1,31 @@
-ButtonGroup is a UI component that displays a set of toggleable buttons and allows a user to select one or several of them. This demo illustrates how to configure the ButtonGroup to enable single and multiple selection.
+ButtonGroup displays a set of two-state buttons and allows users to select one or multiple buttons. This demo illustrates how to configure the ButtonGroup in the following steps:
+
+- Bind ButtonGroup to data.
+
+- Enable or disable multiple selection.
+
+- Customize the ButtonGroup appearance.
+
+## Bind ButtonGroup to Data
+
+Use the [items](/Documentation/ApiReference/UI_Components/dxButtonGroup/Configuration/items/) array to populate ButtonGroup with data. You can specify the following fields in the data objects: 
```

Comment:
I believe we can go by without this article. Mainly because we haven't named them explicitly here.

```suggestion
Use the [items](/Documentation/ApiReference/UI_Components/dxButtonGroup/Configuration/items/) array to populate ButtonGroup with data. You can specify the following fields in data objects: 
```